_posts/2023-09-10-august-rubygems-updates.md

```diff
@@ -0,0 +1,76 @@
+---
+title: August 2023 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at Ruby Central, we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems and RubyGems.org in August.
+
+## RubyGems News
+
+This month in RubyGems, we released [3.4.18](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3418--2023-08-02), [3.4.19](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3419--2023-08-17) and Bundler [2.4.18](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2418-august-2-2023), [2.4.19](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2419-august-17-2023).
+
+
+In RubyGems, we improved the documentation performance by speeding up the build process when updating RubyGems - [#6864](https://github.com/rubygems/rubygems/pull/6864). We added a poller to fetch WebAuthn OTPs - [#6774](https://github.com/rubygems/rubygems/pull/6774), and removed some side effects when unmarshalling old `Gem::Specifications`- [#6825](https://github.com/rubygems/rubygems/pull/6825).
+
+Some other improvements that landed into our repo this month but that are not included in the above releases are:
+
+- optimized memory usage in `Bundler::Settings`, resulting in a faster boot time - [#6884](https://github.com/rubygems/rubygems/pull/6884).
+- added a `file` option to the Ruby method in the Gemfile- [#6876](https://github.com/rubygems/rubygems/pull/6876).
```

Comment:
This one is included in 2.4.19 Bundler release, so it could shift up to that section.